app/organizations.ts

```diff
@@ -99,11 +96,10 @@ export const organizationsStore: Organizations = {
   "Nola-Game-Dev":{
     "organizers": [
       {
-        "name":"Some Guy McPlaceholder",
-        "pfp": "/organizersPfp/ben.webp",
+        "name":"Calder",
+        "pfp": "https://via.placeholder.com/158x158",
         "links": [
-          {"linkedin": "https://www.linkedin.com/in/benjamin-eckel-b025831a3/"},
-          {"email": "benjamin.eckel@gmail.com"}
+          {"website": "https://nolagamedev.wordpress.com/contact/"}
```

Comment:
I think that the URL should be [http://neworleansgamedev.com](http://neworleansgamedev.com) (note, the URL is http-only for some reason...)

They use this url on their Discord and all of their official communication channels so I'm assuming that the wordpress site was just an old site that they used to use.

Reply:
They have an organization-wide email (found on their Discord) if that would be better to put there.
`org@neworleansgamedev.com`

Reply:
Appreciate your help @Aeshus 

I implemented your suggestions and am about to push up. I elected to use the `contact` email address. I reached out to Calder and he gave me a date he would provide me assets and contact info, but that date came and went so I don't want to pester him about it. I will reach out again soon.